### ISSUE TITLE
Close runCommand stdout using communicate

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8640]
|[<img alt="scop's avatar" src="https://avatars.githubusercontent.com/u/109152?s=50" width="50" height="50">](https://github.com/scop)| @scop reported|
|-|-|
|Trac ID|trac#8640|
|Type|enhancement|
|Created|2016-07-20 14:52:19Z|

<details><summary>Searchable metadata</summary>

```
trac-id__8640 8640
type__enhancement enhancement
reporter__scop scop
priority__normal normal
milestone__None None
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__release_management release management
keywords__None None
time__1469026339945649 1469026339945649
changetime__1469075127303414 1469075127303414
version__None None
owner__GitHub__noreply_____ GitHub <noreply@...>
cc__radix
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="scop's avatar" src="https://avatars.githubusercontent.com/u/109152?s=50" width="50" height="50">](https://github.com/scop)<a name="note_2"></a>|@scop commented|
|-|-|

https://github.com/twisted/twisted/pull/368

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_3"></a>|@rodrigc commented|
|-|-|

In changeset 19e77adf865bdc9ac1f086b04fb4505a0740be74
```
#!CommitTicketReference repository="" revision="19e77adf865bdc9ac1f086b04fb4505a0740be74"
Add topfile for #8640
```

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_4"></a>|GitHub set owner to GitHub<br>GitHub set status to `closed`|
|-|-|

In changeset 6a4bbc8a7750e7f2e5a2c2c95347203bdf9a5059
```
#!CommitTicketReference repository="" revision="6a4bbc8a7750e7f2e5a2c2c95347203bdf9a5059"
Merge pull request #374 from twisted/8640-scop-release-stdout

Author: scop
Reviewer: markrwilliams, rodrigc
Fixes: #8640

Close runCommand stdout using communicate
```

**Comment by twisted-trac** [comment]
|[<img alt="markrwilliams's avatar" src="https://avatars.githubusercontent.com/u/704247?s=50" width="50" height="50">](https://github.com/markrwilliams)<a name="note_5"></a>|@markrwilliams commented|
|-|-|

From GitHub:

This looks good. I'll probably open a new ticket for replacing runCommand with subprocess.check_output. Feel free to do that now if you'd like

In the meantime, merge this.
